Makefile

```diff
@@ -16,53 +16,100 @@ $(warning podman detected but 'podman version' failed. \
 endif
 endif
 
-BUILD_CMD:=$(CONTAINER_CMD) build $(BUILD_OPTS)
-PUSH_CMD:=$(CONTAINER_CMD) push $(PUSH_OPTS)
-SHELLCHECK:=shellcheck
```

Comment:
I don't think these changes are necessary.

Reply:
I think those were committed by accident, and they have been undone in latest updates